inlong-agent/agent-common/src/main/java/org/apache/inlong/agent/constant/SnapshotModeConstants.java

```diff
@@ -27,4 +27,12 @@ public class SnapshotModeConstants {
     public static final String EARLIEST_OFFSET = "never";
 
     public static final String SPECIFIC_OFFSETS = "schema_only_recovery";
+
+    public static final String ALWAYS = "always";
```

Comment:
if these `Constants` only used for PostgreSQL, it's better to be added in a independent class.

Reply:
OK， i will move it to independent class.